spec.html

```diff
@@ -150,32 +150,43 @@ <h1>The ElementDescriptor Specification Type</h1>
       <p>
         The <dfn>ElementDescriptor</dfn> is a Record used to represent class elements at runtime.
         Values of the ElementDescriptor type are Record values whose fields are defined as by <emu-xref href="#table-element-descriptor-fields"></emu-xref>.
-        Unless otherwise specified, every field is always present.
+        Unless otherwise specified, every field is optional.
```

Comment:
why this change?

Reply:
Because if [[Kind]] is "internal" every field (except `[[Element]]`) is absent.